### DISCUSSION TITLE
Property 'count' does not exist on type 'object'

### DISCUSSION THREAD

**Discussion by relrelway** [pmndrs/zustand#1043] [Q&A]
Property 'count' does not exist on type 'object'
code:
```ts
const useStore = create((set) => ({
  count: 1,
  inc: () => set((state) => ({ count: state.count + 1 })),
}));
```
<img width="1004" alt="Screen Shot 2022-06-28 at 3 12 00 PM" src="https://user-images.githubusercontent.com/64797361/176129439-a112ca73-121f-4331-834a-6e684f053feb.png">

**Comment by dai-shi** [comment]
Please have a look at Typescript Usage: https://github.com/pmndrs/zustand#typescript-usage

If you need further assistance, let us know. (I'm sure someone can help.)
